torch/_torch_docs.py

```diff
@@ -261,6 +261,11 @@ def merge_dicts(*dicts):
 Performs the element-wise division of :attr:`tensor1` by :attr:`tensor2`,
 multiply the result by the scalar :attr:`value` and add it to :attr:`input`.
 
+.. warning::
+    Integer division with addcdiv is deprecated and will be removed in a future
```

Comment:
as above, tell them what to use instead.

Reply:
Sounds good.